test/test_scc.jl

```diff
@@ -1,32 +1,37 @@
 using Test
 using Mimi
 
-include("../src/main_model.jl")
-include("../src/compute_scc.jl")
-
 m = getpage("RCP4.5 & SSP2", true, true, true)
 run(m)
 uds = [undiscount_scc(m, 2020), undiscount_scc(m, 2040), undiscount_scc(m, 2075)]
 uds_compare = [0.8765, 0.5279, 0.2162] # From Excel PAGE-ICE
 @test uds ≈ uds_compare rtol=1e-4
 
-function checksccs(sccs, sccomp)
+function checksccs(sccs, scccomp; n::Int = 1000)
     sccobs = [quantile(sccs, [.05, .25, .5, .75, .95]); mean(sccs)]
     sccerror = abs.((sccobs - scccomp) ./ [10, 1, 1, 1, 10, 1])
-    sccsd = std(sccs) / sqrt(1000)
+    sccsd = std(sccs) / sqrt(n)
     @test all(sccerror .< 3*sccsd) # average off by < 3x SD
 end
 
+n = 1000
+
 m = getpage("RCP4.5 & SSP2", true, true, true)
-sccs = compute_scc_mcs(m, 1000, year=2020)
-checksccs(sccs, [37.031, 89.742, 161.868, 299.439, 847.020, 261.739])
+sccs = compute_scc_mcs(m, n, year=2020)
+# TODO - get the baseline values to test against!
+# checksccs(sccs, [37.031, 89.742, 161.868, 299.439, 847.020, 261.739]; n = n)
```

Comment:
@jrising I don't think this file has worked in a while, given there was a bug in the syntax and it probably was holding old PAGE2009 values but I'm really not sure? 

We already check the mcs version of the scc in `test_mcs.jl` in the same way we did in PAGE2009, so this looks like a similar type of test but with more scenarios?  I don't know if we really need to test it here as well, and perhaps the protocol we used to do it in `test_mcs.jl` is more up to date, regardless I didn't want to delete this test without asking you your opinion!

Reply:
alternatively we leave that for a new PR, and merge this and #74 first, since this really isn't related to those

Reply:
^ I'd probably recommend that to keep things clean and moving along

Reply:
Oh, you're probably right that it hadn't been updated. PAGE-ICE didn't publish SCC values, so there was nothing to benchmark against. I can clean this up later, but let's not worry about it for what you're doing now.

Reply:
Great, we can leave it how it is for this merge ... it'll test that the functionality runs but won't compare it to anything